deploy/manifests/kip/overlays/mac/provider.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+cloud:
+  aws:
+    accessKeyID: ""
+    secretAccessKey: ""
+etcd:
+  internal:
+    dataDir: /opt/kip/data
+cells:
+  bootImageSpec:
+    owners: 689494258501
+    filters: name=elotl-kip-*
+    macfilters: name=elotl-kipmac-*
```

Comment:
Did the multi-image PR end up using `macfilters`? That will be a problem once ARM Mac instances become available.

Reply:
This will have to be changed to x86_64_mac_filters. See #203